user_guide_src/source/general/helpers/008.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+// app/Helpers/info_helper.php
+
+/**
+ * Returns CodeIgniter's version.
+ */
+function ci_version(): string
+{
+    return CodeIgniter::CI_VERSION;
```

Comment:
```suggestion
    return CodeIgniter\CodeIgniter::CI_VERSION;
```
OR use CodeIgniter\CodeIgniter;

Reply:
Done.

Reply:
Thanks @kenjis.